source/reference/minio-kubectl-plugin.rst

```diff
@@ -331,33 +359,43 @@ Expand a MinIO Tenant
 .. mc-cmd:: tenant expand
    :fullpath:
 
-   Adds a new zone to an existing MinIO Tenant.
+   Extends the total capacity of a MinIO Tenant by adding a new Pool. A Pool
+   consists of a set of MinIO Pods and their associated storage resources.
 
-   The command creates the new zone using the 
-   :minio-git:`latest release <minio/minio/releases/latest>` of ``minio``:
+   :mc-cmd:`~kubectl minio tenant expand` always uses the latest stable 
```

Comment:
@nitisht @Alevsk I have a feeling this isn't true.

Reply:
`expand` command doesn't change the image. It will keep running the current image.

Reply:
Done,thanks.